lib/rom/sql/extensions/postgres/commands.rb

```diff
@@ -12,22 +12,26 @@ module Create
           def insert(tuples)
             tuples.map do |tuple|
               relation.dataset.returning(*relation.columns).insert(tuple)
-            end.flatten
+            end.flatten.map { |t| relation.read_schema[t] }
```

Comment:
@solnic we have to do this because command result must piped through `read_schema`